### ISSUE TITLE
Integrate Qiskit server in C# script

### ISSUE THREAD

**Issue by HuangJunye** [HuangJunye/QPong-Unity#54]
Do you think it’s a good idea to make a simple quantum simulator in C# instead of calling Qiskit over html? I think it will make the game standalone and more portable. 

It should not be too difficult to make that. James Wootton made a simplified version in python https://github.com/quantumjim/aether I can rewrite it in C#

**Comment by HuangJunye** [comment]
What do you think? @boland25 @filemaster

**Comment by filemaster** [comment]
I think you don't need to rewrite all simulator, you can try to simulate the Quantum Computer behaviour for the gates the game is using now.

**Comment by HuangJunye** [comment]
Yes, definitely. I will just write the essential things that are needed for the game. James Wootton's aether is a very simple quantum simulator (~100 lines).
